src/content/docs/guides/Web development.mdx

```diff
@@ -416,6 +750,46 @@ Teraz jest bardzo jasne, że po kliknięciu na mięsień, zostaje też zupdate
 
 Ten przykład jeszcze nie jest taki zły, przy 1 useEffectcie dość łatwo się połapać co się dzieje, przy 11 robi się już kłopot i raz u mnie w pracy przez coś takiego padła produkcja 😭
 
+### Nieużywanie `useMemo`
```

Comment:
to jest zła praktyka, nie powinno się używać useMemo jeśli nie ma `expensive computation` a I tak w takich przypadkach powinno być dodawane reaktywnie, bo programiści są bardzo slabi w określaniu co to jest `expensive computation`

https://maxrozen.com/understanding-when-use-usememo

nadużywanie useMemo I useCallback to jeden z typowych błędów początkujących, możesz o tym dopisać 😭

zapraszam też do dokumentacji reacta
![Screenshot_20241128-075409.png](https://github.com/user-attachments/assets/e93348a7-26b6-4295-ab4a-c68203f85dfe)
https://react.dev/reference/react/useMemo

Reply:
ugotowane, much better, bardzo dziękuję